<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QMainWindow" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WareHouse Sign In/Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	color: #000000;
	border-color: #000000;

}
QStackedWidget
{
	background: transparent;
}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #379683;
	color: rgb(237, 245, 225);
    font-family: &quot;Arial&quot;;
    font-size: 12px;
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: #379683;
	margin: 2px;

}

QPushButton:hover {
    background-color: #2d6a5f; 
}

QPushButton:pressed {
    background-color: #266057;
}

/*-----QLineEdit-----*/
QLineEdit
{
    font-family: &quot;Arial&quot;;
	background-color: rgb(237, 245, 225);
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-radius: 4px;
	border-color: #000000;
	font-family: &quot;Azonix&quot;;

}

QToolButton {
    background-color: transparent; 
    border: none; 
}

QLabel {
	background: transparent;
    font-family: &quot;Arial&quot;;
    font-size: 12px;
    color: #333333;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="LoginPagesWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="LoginPage">
     <widget class="QToolButton" name="LoginIconButton_Log">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>85</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="RegIconButton_Log">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>155</y>
        <width>22</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="BackGround">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="InfoLabel_Log">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="PassLabel_Log">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>135</y>
        <width>209</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>265</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginLineEdit_Log">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>89</y>
        <width>209</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QToolButton" name="showPassButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/unshowpass.png</normaloff>icons/unshowpass.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ShowPassLabel">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>180</y>
        <width>176</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Отображать пароль?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PassLineEdit_Log">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>155</y>
        <width>209</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RegButton_Log">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>265</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
     <widget class="QLabel" name="LoginLabel_Log">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>69</y>
        <width>209</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <zorder>BackGround</zorder>
     <zorder>line_3</zorder>
     <zorder>LoginIconButton_Log</zorder>
     <zorder>RegIconButton_Log</zorder>
     <zorder>InfoLabel_Log</zorder>
     <zorder>PassLabel_Log</zorder>
     <zorder>LoginButton</zorder>
     <zorder>LoginLineEdit_Log</zorder>
     <zorder>showPassButton</zorder>
     <zorder>ShowPassLabel</zorder>
     <zorder>PassLineEdit_Log</zorder>
     <zorder>RegButton_Log</zorder>
     <zorder>LoginLabel_Log</zorder>
    </widget>
    <widget class="QWidget" name="RegisterPage">
     <widget class="QToolButton" name="RegIconButton_Reg">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>22</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="LoginLabel_Reg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>189</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginLineEdit_Reg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BackToLoginButton">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>241</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QToolButton" name="LoginIconButton_Reg">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>22</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="InfoLabel_Reg">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>211</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="PassLabel_Reg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>189</width>
        <height>14</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PassLineEdit_Reg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="TryRegButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
     <widget class="QLabel" name="BackGround_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>BackGround_2</zorder>
     <zorder>RegIconButton_Reg</zorder>
     <zorder>line_2</zorder>
     <zorder>LoginLabel_Reg</zorder>
     <zorder>LoginLineEdit_Reg</zorder>
     <zorder>BackToLoginButton</zorder>
     <zorder>LoginIconButton_Reg</zorder>
     <zorder>InfoLabel_Reg</zorder>
     <zorder>PassLabel_Reg</zorder>
     <zorder>PassLineEdit_Reg</zorder>
     <zorder>TryRegButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
